pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -133,6 +137,27 @@ func EnforceHostNetwork() bool {
 	return cephv1.EnforceHostNetwork()
 }
 
+func SetRevisionHistoryLimit(data map[string]string) {
+	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, revisionHistoryLimitDefaultValue)
+	var numval int64 = 0
+	if strval != "" {
+		var err error
+		numval, err = strconv.ParseInt(strval, 10, 32)
+		if err != nil {
+			logger.Warningf("failed to parse value %q for %q. assuming default value 3.", strval, revisionHistoryLimitSettingName)
+			return
+
+		}
+	}
+	limit := int32(numval)
+	revisionHistoryLimit = &limit
```

Comment:
Lets move this to inside the if condition when strval is not empty

Reply:
good point. done